packages/client/hmi-client/src/components/data-explorer/search-item.vue

```diff
@@ -76,6 +202,17 @@ const formatDetails = () =>
 	margin: 0.25rem 0 0.5rem 0;
 }
 
+.title,
+.details,
+.description,
+.parameters,
+.features {
+	display: -webkit-box;
+	-webkit-box-orient: vertical;
+	-webkit-line-clamp: 1;
+	overflow: hidden;
+}
```

Comment:
This should be moved to `src/assets/css/styles.css`

Reply:
Others used these class names in other components and were defined differently. Are you sure about this?

Reply:
Sorry, I haven't seen this being classes... my bad!